tests/ui/type-alias-impl-trait/wf-in-associated-type.rs

```diff
@@ -1,14 +1,15 @@
 // WF check for impl Trait in associated type position.
 //
-// revisions: pass fail
+// revisions: pass pass_next fail
```

Comment:
Doesn't this also need a
```rust
//[pass_next] compile-flags: -Ztrait-solver=next
```

(sorry if I'm missing something)

Reply:
yea good catch